src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
?

Reply:
I don't think it's necessary
I'll remove it